src/main/java/com/books/servlets/BookList.java

```diff
@@ -0,0 +1,41 @@
+
+package com.books.servlets;
+
+
+import com.books.services.BookService;
+import com.books.storage.concrete.BookRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+
+public class BookList extends HttpServlet {
```

Comment:
Please, follow naming convention as you did in another ones servlets. E.g. BookListServlet. It better describes what does this class do.